### ISSUE TITLE
Translation of certain elements in the repeat rule properties are not taken into account

### ISSUE THREAD

**Issue by ed-kffein** [solspace/craft3-calendar#47]
I'm trying to translate the plugin content in french following the documented procedure https://docs.solspace.com/craft/calendar/v3/setup/translating.html 
Some elements, included in the original vendor/solspace/craft3-calendar/src/translations/en-US/calendar.php file are not taken into account when translated. 

**Steps to reproduce**

1. Copy vendor/solspace/craft3-calendar/src/translations/en-US/calendar.php to translations/fr/calendar.php
2. Translate the elements. Affected entries are : 
```
'End repeat' => 'Fin de la répétition',
'Except on' => 'Excepté le',
```
3. On the event edition page, the elements are still displayed in english : 
![image](https://user-images.githubusercontent.com/78917206/123850401-fdadc180-d8e7-11eb-933b-0fd03286d545.png)

**Expected behavior**
The text should be translated

**Craft & Plugin Info :**

- Craft Version: 3.6.17
- Calendar Version: 3.3.0
- Calendar Edition: PRO
- Fresh Install or Upgrade: Fresh Install

**Comment by kjmartens** [comment]
Thanks, we'll look into it. :)

**Comment by kjmartens** [comment]
This is now fixed in Calendar 3.3.2. You'll need to use `End Repeat` and `Except On` instead though. :)
